src/tr1/game/item_actions/flood.c

```diff
@@ -23,5 +24,5 @@ void ItemAction_Flood(ITEM *item)
         Sound_Effect(SFX_WATERFALL_BIG, &pos, SPM_NORMAL);
     }
 
-    g_FlipTimer++;
+    Room_SetFlipTimer(++flip_timer);
```

Comment:
Perhaps Room_IncrementTimer(int32_t frames);?

Reply:
Thanks, I've introduced this and tidied things up. In some cases we check frame numbers so that's been tweaked to take into account the local copies of the value, which are also now const for clarity.